packages/block-library/src/post-title/edit.js

```diff
@@ -138,41 +143,80 @@ export default function PostTitleEdit( {
 						/>
 					</BlockControls>
 					<InspectorControls>
-						<PanelBody title={ __( 'Settings' ) }>
-							<ToggleControl
-								__nextHasNoMarginBottom
+						<ToolsPanel
+							label={ __( 'Settings' ) }
+							resetAll={ () => {
+								setAttributes( {
+									isLink: false, // Default to true
```

Comment:
```suggestion
									isLink: false,
```

I didn't understand the intention of this comment, but I think this comment can be deleted.